snakedeploy/collect_files.py

```diff
@@ -26,30 +30,26 @@ def collect_files(config_sheet_path: str):
             raise UserError(f"No input pattern in config sheet matches {item}.")
         elif len(matches) > 1:
             raise UserError(f"Item {item} matches multiple input patterns.")
-        else:
-            match = matches[0]
-            pattern = match.rule.glob_pattern.format(
-                **{
-                    key: autoconvert(value)
-                    for key, value in match.match.groupdict().items()
-                }
-            )
-            files = sorted(glob(pattern))
-            if files:
-                print(item, *files, sep="\t")
-            else:
-                raise UserError(
-                    f"No files were found for {item} with pattern {pattern}."
-                )
+
+        match = matches[0]
+        pattern = match.rule.glob_pattern.format(
```

Comment:
If we don't use pandas the glob_pattern attribute is no longer there. This all becomes a bit more cumbersome. Since pandas is a soft Snakemake dependency anyway, I think it won't hurt to keep the code simpler and easier to maintain and use pandas. Also, we might use it for other functionality in the future.

Reply:
ahh okay, I'll change it back :) Give me a few minutes!

Reply:
okay, pandas is added back!